catroid/src/androidTest/java/org/catrobat/catroid/test/ui/BaseExceptionHandlerTest.java

```diff
@@ -46,14 +44,10 @@
 	@Before
 	public void setUp() throws Exception {
 		super.setUp();
-		context = InstrumentationRegistry.getTargetContext();
-		sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context);
-		editor = sharedPreferences.edit();
-		editor.clear();
-		editor.putBoolean(SettingsActivity.SETTINGS_CRASH_REPORTS, true);
-		editor.commit();
-		CrashReporter.setIsCrashReportEnabled(true);
-		CrashReporter.initialize(context);
+		CrashReporterTest.setUpMethod();
+		context = CrashReporterTest.findContext();
+		sharedPreferences = CrashReporterTest.getSharedPreferences();
+		editor = CrashReporterTest.getEditor();
```

Comment:
That's a little over the top imo. I guess you thought that as well when creating the getters.
I think, the best code that can be shared is a util method which enables and initialize crash reporting. However, I'm not sure if it's worth it. What do you think?